content/working_with/integration/jenkins-plugin.md

```diff
@@ -34,12 +34,12 @@ In certain environments, access to the public Jenkins Update Center is restricte
 to cope with it (such as using a custom Update Center configuration file), which are described in
 Jenkins' documentation.
 
-If you would like to install the Cloudify Jenkins plugin directly, you can download the plugin file
+If you would like to install the {{< param product_name >}} Jenkins plugin directly, you can download the plugin file
 (with `.hpi` extension) from Jenkins' Maven artifact repository: https://repo.jenkins-ci.org/
 
 Our plugin's Group ID is `io.jenkins.plugins` and the Artifact ID is `cloudify`. You can therefore
```

Comment:
That is probably something which should be changed on the plugin side, right?

Reply:
yes